packages/util/src/string.ts

```diff
@@ -109,3 +109,22 @@ export function utilSafeString(str) {
 export function utilUniqueString(val) {
   return 'ideditor-' + utilSafeString(val.toString()) + '-' + new Date().getTime().toString();
 }
+
+// Returns a comparator function for sorting strings alphabetically in ascending order,
+// regardless of case or diacritics.
+// If supported, will use the browser's language sensitive string comparison, see:
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator
+export function utilSortString(locale = 'en'): (x: string, y: string) => number {
+  if (typeof Intl === 'object' && 'Collator' in Intl) {
+      return (new Intl.Collator(locale, {
+          sensitivity: 'base',
+          numeric: true
+      })).compare;
+  } else {
+      return (a, b) => {
+          a = removeDiacritics(a.toLowerCase());
+          b = removeDiacritics(b.toLowerCase());
```

Comment:
Some languages would normally expect `accent` sensitivity for sorted lists, not just `base`.  I did suggest diacritic folding in https://github.com/openstreetmap/iD/pull/8826#discussion_r757821753, but the language-aware sorting offered by `Intl.Collator` probably makes that unnecessary for anything other than searching and filtering use cases. For the imagery layers in openstreetmap/iD#8826, it’s moot because so few of the items are actually localized, but it does matter for other lists that are more thoroughly localized, like the validator rules in the Issues panel.

Reply:
Can you elaborate on this? I'm going to accept the PR regardless (we can always remove it later it if it turns out to be not useful), but it sounds like folded sorting is not actually the behavior you want in this case, nor what anyone wants in most list displays.